airflow/api_connexion/openapi/v1.yaml

```diff
@@ -1353,6 +1366,14 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/ImportError'
+    
+    Metadatabase:
+      type: object
+      description: Returns the status of the metadatabase
+      properties:
+        statuses:
+          type: string
```

Comment:
Is there a reason why we have statuses in one place and status in another?

Reply:
Here is the change `b3ec805`